frontend/src/components/review/ExistingComment.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useContext } from "react";
+import { useMutation } from "@apollo/client";
+import { Flex, Box, Button } from "@chakra-ui/react";
+import AuthContext from "../../contexts/AuthContext";
+import WIPComment from "./WIPComment";
+import {
+  UPDATE_COMMENT_BY_ID,
+  UpdateCommentResponse,
+} from "../../APIClients/mutations/CommentMutations";
+
+export type ExistingCommentProps = {
```

Comment:
dont forget about displaying the time! 
![image](https://user-images.githubusercontent.com/15113249/133848772-96f1ea53-77d7-4565-935d-e8394e801eee.png)